src/libs/ReportActionsUtils.js

```diff
@@ -174,6 +175,34 @@ function getLastVisibleMessageText(reportID, actionsToMerge = {}) {
         .substring(0, CONST.REPORT.LAST_MESSAGE_TEXT_MAX_LENGTH);
 }
 
+/**
+ * Filter out any reportActions which should not be displayed
+ * Then, sort by their created timestamp first, and reportActionID second
+ * This is a combination of getSortedReportActions() and filterReportActionsForDisplay()
+ *
+ * @param {Array} reportActions
```

Comment:
`reportActions` is an object, not an Array

```suggestion
 * @param {Object} reportActions
```